src/main/java/org/example/mappers/JobRoleToCSV.java

```diff
@@ -0,0 +1,55 @@
+package org.example.mappers;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import com.opencsv.CSVWriter;
+import org.example.models.JobRoleInfoResponse;
+
+public final class JobRoleToCSV {
+    private JobRoleToCSV() {
+        throw new UnsupportedOperationException("This is a utility class "
+                + "and cannot be instantiated");
+    }
+
+    public static void writeJobRoleForPipeSeparatorCSV(
```

Comment:
Need to change name as you are no longer separating by pipes

Reply:
Changed